tests/subsys/fs/multi-fs/src/main.c

```diff
@@ -10,28 +10,36 @@
 #include "test_littlefs.h"
 #include "test_fs_shell.h"
 
-void test_main(void)
+ZTEST(multi_fs_fat, test_multi_fs_fat)
 {
```

Comment:
Can we add a comment explicitly calling these out as order-dependent? After that LGTM.

Reply:
I have modified it. Please have a look again.